pkg/github/projects.go

```diff
@@ -32,8 +32,9 @@ func ListProjects(t translations.TranslationHelperFunc) inventory.ServerTool {
 			Name:        "list_projects",
 			Description: t("TOOL_LIST_PROJECTS_DESCRIPTION", `List Projects for a user or organization`),
 			Annotations: &mcp.ToolAnnotations{
-				Title:        t("TOOL_LIST_PROJECTS_USER_TITLE", "List projects"),
-				ReadOnlyHint: true,
+				Title:         t("TOOL_LIST_PROJECTS_USER_TITLE", "List projects"),
+				ReadOnlyHint:  true,
+				OpenWorldHint: jsonschema.Ptr(true),
```

Comment:
Can projects be open world? I think they might only be available inside paying orgs or enterprises although I could be wrong there.